brave/brave-ios
# Fix #1923: Shields & Settings Design Update

Can be squash merged

## Summary of Changes


This pull request fixes #1923 

## Submitter Checklist:


## Test Plan:

- Verify shields act as usual
- Verify shields icon on NTP is disabled like desktop
- Verify global shields act as usual
- Verify all settings toggles act as normal (a number of them have moved)
- Verify new design (themes, desktop parity, etc.)
- Verify new report a site functionality (may require asking @aekeus) 

## Screenshots:

<details>
<summary><strong>Shields On</strong></summary>

| Basic | Advanced | Scrolled | Info |
| --- | --- | --- | --- |
| ![Simulator Screen Shot - iPhone 11 Pro - 2020-08-31 at 17 59 50](https://user-images.githubusercontent.com/529104/91773061-080e0a80-ebb4-11ea-927d-eaf69dffcd69.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-08-31 at 17 59 53](https://user-images.githubusercontent.com/529104/91773068-0c3a2800-ebb4-11ea-90cb-45438ffaf2d9.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-08-31 at 17 59 55](https://user-images.githubusercontent.com/529104/91773075-0f351880-ebb4-11ea-82e8-f0a7f7ece50e.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-08-31 at 17 59 57](https://user-images.githubusercontent.com/529104/91773083-10fedc00-ebb4-11ea-9da7-d8d04b78234a.png) |

</details>

<details>
<summary><strong>Shields Off</strong></summary>
 
| Basic | Report Site | Reported |
| --- | --- | --- |
| ![Simulator Screen Shot - iPhone 11 Pro - 2020-08-31 at 18 00 01](https://user-images.githubusercontent.com/529104/91773121-27a53300-ebb4-11ea-8181-eaad8d229512.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-08-31 at 18 00 04](https://user-images.githubusercontent.com/529104/91773128-2a078d00-ebb4-11ea-94e9-717b7eff45b0.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-08-31 at 18 00 06](https://user-images.githubusercontent.com/529104/91773133-2c69e700-ebb4-11ea-8d56-4cd774c06c54.png) |

</details>

<details>
<summary><strong>Dark Mode</strong></summary>

<img width=300 src=https://user-images.githubusercontent.com/529104/91773144-325fc800-ebb4-11ea-90c3-9e65c161f1f6.png>

</details>

<details>
<summary><strong>Settings</strong></summary>

| Base | Scrolled |
| --- | --- |
| ![Simulator Screen Shot - iPhone 11 Pro - 2020-08-31 at 18 04 51](https://user-images.githubusercontent.com/529104/91773330-91254180-ebb4-11ea-885d-2204400353b6.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-08-31 at 18 05 00](https://user-images.githubusercontent.com/529104/91773334-93879b80-ebb4-11ea-87f1-b71d766010e3.png) |

| Shields & Privacy | Scrolled |
| --- | --- |
| ![Simulator Screen Shot - iPhone 11 Pro - 2020-08-31 at 18 05 14](https://user-images.githubusercontent.com/529104/91773364-a39f7b00-ebb4-11ea-902a-c8373b2cd2e4.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-08-31 at 18 05 16](https://user-images.githubusercontent.com/529104/91773371-a601d500-ebb4-11ea-9168-a2dfe08e2f64.png) |


</details>

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
